root/treeformula/stl/execReferenceReturnType.C

```diff
@@ -7,6 +7,9 @@ v.emplace_back("a","b");
 t.Fill();
 t.Scan("v@.at(0).GetName()");
 auto res = t.Scan("v@.at(0).fName");
-if (res == 1) return 0;
-else return 1;
+if (res == 1) {
```

Comment:
> Cling's continuation does not support this syntax. 

Do we have a JIRA ticket following this mis-use and/or documentation that it will not be supported?